include/build_config/SDL_build_config_ios.h

```diff
@@ -196,16 +196,13 @@
 #define SDL_PLATFORM_SUPPORTS_METAL 0
 #endif
 
-#if SDL_PLATFORM_SUPPORTS_METAL
-#define SDL_VIDEO_RENDER_METAL  1
-#endif
-
-#if SDL_PLATFORM_SUPPORTS_METAL
-#define SDL_VIDEO_VULKAN 1
-#endif
-
 #if SDL_PLATFORM_SUPPORTS_METAL
 #define SDL_VIDEO_METAL 1
+#define SDL_GPU_METAL 1
```

Comment:
The sorting hurts my brain. Can you either sort alphabetically or by subsystem?

Reply:
I based this off of the macOS build config file, which [orders by API](https://github.com/libsdl-org/SDL/blob/54836050c9893bce3ee2037a6b2b0c4fbda5d43e/include/build_config/SDL_build_config_macos.h#L238). I've switched it now to sorting by subsystem.